src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
@MattGertz hmm... okay.

but just FYI. the reason I added the infobar at the first place was that, I killed service hub process as I would normally do for code lens, and it crashed VS right away. and that felt too much. if we believe that user would not know connection between service hub process and VS as you and other said above, and they kills some process they don't like and suddenly it crashes VS seems a bit too much for me.

anyway, I will make it crash.